packages/contracts/test/test-utils/wrapper/index.ts

```diff
@@ -0,0 +1,196 @@
+  PLUGIN_REPO: 'src/framework/plugin/repo/PluginRepo.sol:PluginRepo',
+  PLUGIN_REPO_V1_0_0:
+    '@aragon/osx-v1.0.1/framework/plugin/repo/PluginRepo.sol:PluginRepo',
+  DAO_REGISTRY: 'src/framework/dao/DAORegistry.sol:DAORegistry',
+  DAO_REGISTRY_V1_0_0:
+    '@aragon/osx-v1.0.1/framework/dao/DAORegistry.sol:DAORegistry',
+  PLUGIN_REPO_REGISTRY:
+    'src/framework/plugin/repo/PluginRepoRegistry.sol:PluginRepoRegistry',
+  PLUGIN_REPO_REGISTRY_V1_0_0:
+    '@aragon/osx-v1.0.1/framework/plugin/repo/PluginRepoRegistry.sol:PluginRepoRegistry',
+  ENS_SUBDOMAIN_REGISTRAR:
+    'src/framework/utils/ens/ENSSubdomainRegistrar.sol:ENSSubdomainRegistrar',
+  ENS_SUBDOMAIN_REGISTRAR_V1_0_0:
+    '@aragon/osx-v1.0.1/framework/utils/ens/ENSSubdomainRegistrar.sol:ENSSubdomainRegistrar',
+  MERKLE_DISTRIBUTOR:
+    'src/plugins/token/MerkleDistributor.sol:MerkleDistributor',
+  MERKLE_DISTRIBUTOR_V1_0_0:
+    '@aragon/osx-v1.0.1/plugins/token/MerkleDistributor.sol:MerkleDistributor',
+  MERKLE_MINTER: 'src/plugins/token/MerkleMinter.sol:MerkleMinter',
+  MERKLE_MINTER_V1_0_0:
+    '@aragon/osx-v1.0.1/plugins/token/MerkleMinter.sol:MerkleMinter',
+  ADDRESSLIST_VOTING:
+    'src/plugins/governance/majority-voting/addresslist/AddresslistVoting.sol:AddresslistVoting',
+  ADDRESSLIST_VOTING_V1_0_0:
+    '@aragon/osx-v1.0.1/plugins/governance/majority-voting/addresslist/AddresslistVoting.sol:AddresslistVoting',
+  TOKEN_VOTING:
+    'src/plugins/governance/majority-voting/token/TokenVoting.sol:TokenVoting',
+  TOKEN_VOTING_V1_0_0:
+    '@aragon/osx-v1.0.1/plugins/governance/majority-voting/token/TokenVoting.sol:TokenVoting',
+  MULTISIG: 'src/plugins/governance/multisig/Multisig.sol:Multisig',
+  MULTISIG_V1_0_0:
+    '@aragon/osx-v1.0.1/plugins/governance/multisig/Multisig.sol:Multisig',
+};
+
+export type DeployOptions = {
+  initArgs?: any[]; // initialize function arguments in case `withProxy` is set to true.
+  args?: any[]; // constructor arguments
+  withProxy?: boolean;
+  proxySettings?: {
+    type?: 'uups' | 'transparent' | 'beacon' | undefined;
+    initializer?: string;
+  };
+};
+
+export interface NetworkDeployment {
+  deploy(artifactName: string, args: any[]): any;
+  getCreateAddress(sender: string, nonce: BigNumberish): string;
+  getNonce(
+    sender: string,
+    type?: 'Deployment' | 'Transaction'
+  ): Promise<number>;
+  encodeFunctionData(
+    artifactName: string,
+    functionName: string,
+    args: any[]
+  ): Promise<string>;
+  deployProxy(
+    deployer: number,
+    artifactName: string,
+    options: DeployOptions
+  ): Promise<Contract>;
+  upgradeProxy(
+    upgrader: number,
+    proxyAddress: string,
+    newArtifactName: string,
+    options: DeployOptions
+  ): Promise<Contract>;
+}
+
+export class Wrapper {
+  network: NetworkDeployment;
+
+  constructor(_network: NetworkDeployment) {
+    this.network = _network;
+  }
+
+  static async create(networkName: string, provider: providers.BaseProvider) {
```

Comment:
Can we add an explanation on what this function does and why this is necessary?
I.e., why do we need to send 0.5 ETH from signer[0] to signers 10 to 19?

Reply:
Added it - https://github.com/aragon/osx/pull/596/commits/e0bc39b745c48ebcad41fb48bfb3200831964440 (see hardhat.config.ts as well for zksynclocal testnet network)

Reply:
We only use `signer[10]` once. Maybe rewriting this one test is enough?